posts/2021-10-05-configurable-response-headers-21.0.0.11-beta.adoc

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: "Configurable Response Headers and more exciting new features in Open Liberty 21.0.0.11-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Configurable Response Headers and more exciting new features in Open Liberty 21.0.0.11-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.11-beta provides new configurable response headers, offering more granular control over response headers, as well as important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.
+blog_description: Open Liberty 21.0.0.11-beta provides new configurable response headers, offering more granular control over response headers, as well as important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Configurable Response Headers and more exciting new features in Open Liberty 21.0.0.11-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 21.0.0.11-beta provides new configurable response headers, offering more granular control over response headers, as well as important changes to the Jakarta EE convenience features, with the removal of Jakarta EE 9.0 features in the transition to Jakarta EE 9.1.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9.1 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9.1 Beta Features>>: a lightweight package that contains only the Jakarta EE 9.1 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<response_headers, Configurable Response Headers>>
+
+[#response_headers]
+=== Configurable Response Headers
+
+You can now configure Open Liberty (21.0.0.11-beta) to modify response headers. The available configuration options allow for headers to be appended, for existing headers to be overwritten, for missing headers to be added, and for undesired headers to be removed from all responses being serviced by an HTTP endpoint. This configuration offers more granular control over response headers, which offers a solution to modifying headers without the need to modify existing applications, filters, or otherwise. 
```

Comment:
suggested edit for concision;

 This configuration offers more granular control over response headers, which offers a solution to modifying headers without the need to modify existing applications, filters, or otherwise. 
--->
 This granular control over response headers means you can modify headers without having to modify existing applications or filters.